features/FxA-56-email-deliverability/README.md

```diff
@@ -1,66 +1,160 @@
+Users regularly report email deliverability issues with Firefox Accounts,
+and we do not have sufficient visibility into
+the common causes of email bounces
+to detect or debug them.
 
 ----
 
 ## Outcomes
 
-To have better metrics and logging around our email delivery, with
-the goal of improving email deliverability afterwards.
+To have better metrics and logging around our email delivery,
+so we can judge the effectiveness of any attempts to improve
+our delivery rate.
+
+To try one simple tactic for improving email deliverability
+and measure whether it is successful.
 
 ## Hypothesis
 
-If we add metrics to email services in FxA then
-**we will have a chance to improve the email deliverability** of the
-authentication system.
+If we add email deliverability metrics to FxA then
+**we can take a data-driven approach to improving email deliverability**
+of the authentication system.
 
-If we stop sending additional email messages to bounced emails addresses then our "complaint rate" with popular email providers will decrease and improve our deliverability.
+If we avoid sending additional emails to addresses that are known to bounce then
+**our "sender reputation" wth popular email providers will improve**
+and hence our email deliverability will improve.
 
 
 ## Metrics
 
-The following metrics should help us understand and debug email deliverability:
+The following metrics will be used
+to understand and debug email deliverability:
 
-* Flow events for email bounces
-* Current bounce rate
-* A dashboard that lists common
+### Completion Rate for Email Flows
 
-----
+For flows that result in an email being sent,
+we will produce a flow funnel graph
+showing the rate of dropoff for the following events:
 
-## Detailed design
+* The email being successfully delivered by SES
+* The link in the email being clicked
+* The flow completing
+
+The graph will look something like this:
+
+![Mock graph for completion rate of email flows](email_flow_completion_rate.jpg)
+
+This will help us to know what percentage of flows
+fail to due email delivery issues.
+
+Future attempts to improve email deliverability
+can be judged successful
+if they improve this completion rate.
+
+### Lost Mail Rate for Email Flows
+
+For flows that result in an email being sent,
+we will produce a graph showing the percentage for which:
 
-There are different parts that will help us solve our email deliverability problems:
+* We receive a successful delivery notification from SES
+* We receive a bounce notification from SES
+* we receive no further information about delivery
 
-* Adding flow events for bounces
+The graph will look something like this:
 
-The plan is to add flow event metrics to our existing bounce tracking scripts in the `fxa-auth-server`. This approach will help us create a funnel metrics graph.
+![Mock graph for lost mail rate of email flows](lost_mail_rate.jpg)
```

Comment:
Not sure I understand the difference between `link clicked` and `flow completed`. Seems like they would be the same value?